protocol/0012-POSR-position_resolution.md

```diff
@@ -22,147 +22,48 @@ Any trader that has insufficient collateral to cover their margin liability is r
 
-#### STEP 2
+- Position evolution evaluation is never carried out during an auction. Leaving auction of any type (including opening auction) triggers position resolution evaluation and restarts the timer. Subsequent evaluations are triggered once the elapsed time since last evaluation is greater than `market.positionResolutionFrequency` or when market exits some other auction (whichever comes first).
 
-`NetOutstandingLiability = 5 - 4 + 2 = 3`
+- If position resolution evaluation happens in the same block as the mark price is updated and/or margin levels are re-evaluated then it should happen as last of the three operations.
 
-#### STEP 3
+- If a party marked as distressed has the cross marginging mode selected for the market being considered:
 
-```json
-LiquiditySourcingOrder: {
-  type: 'market',
-  direction: 'sell',
-  size: 3
-}
+  - all their open orders on that market get cancelled without releasing any collateral from the margin account,
+  - their margin level then gets recalculated to account only for the open volume they hold in a market (TODO: if we generalised Tom's notion of an "order margin" for the isolated margin mode to all margin modes then this step would be redundant)  
+  - if party's margin account balance is now above or equal to their latest maintenance margin level then such party is no longer marked as "distressed".
 
-LiquiditySourcingTrade1: {
-  buyer: Trader4,
-  seller: Network,
-  size: 2,
-  price: 120,
-  type: 'liquidity-sourcing'
-}
+- If party's is marked as distressed then:
 
-LiquiditySourcingTrade2: {
-  buyer: Trader5,
-  seller: Network,
-  size: 1,
-  price: 100,
-  type: 'liquidity-sourcing'
+  - their open volume get's added to network's open volume for that market,
+  - party's open volume in that market gets set to 0,
+  - the full amount of party's margin account balance gets forcibly transferred to market's insurance pool.
 
-}
+This concludes the position resolution from party's perspective.
 
-```
+#### Managing network's position
 
-#### STEP 4
+Whilst network has a non-zero position in a given market it's treated as any other party with market's insurance in that market acting as its margin account and with and exception that margin search, margin release or liquidation are never attempted on a network party.
 
-Close out trades are generated with the distressed traders
+Whenever party has a non-zero position it attempts to unload it using an [immediate or cancel](./0014-ORDT-order_types.md) limit order. If the the network has a `long` position it will submit a `sell` order. If it has a short position it will submit a buy order. The size of that order is chosen according to the liquidation strategy which forms a part of the marke's configuration. The strategy can be updated at any point whilst market is active with a market change [governance vote](./0028-GOVE-governance.md#2-change-market-parameters).
```

Comment:
```suggestion
Whenever the network party has a non-zero position it attempts to unload it using an [immediate or cancel](./0014-ORDT-order_types.md) limit order. If the the network has a `long` position it will submit a `sell` order. If it has a short position it will submit a buy order. The size of that order is chosen according to the liquidation strategy which forms a part of the marke's configuration. The strategy can be updated at any point whilst market is active with a market change [governance vote](./0028-GOVE-governance.md#2-change-market-parameters).
```